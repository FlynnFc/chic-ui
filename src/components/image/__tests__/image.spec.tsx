import Image from '..';
import { render } from 'test/utils';

describe('Image Component', () => {
  it('Renders Correctly', () => {
    const ImageGroup = render(
      <>
        <Image src="cat1.png" width="160px" height="130px" />
        <Image src="cat2.png" width="160px" height="130px" />
        <Image src="cat1.png" width="160px" height="130px" rounded />
        <Image src="cat2.png" width="160px" height="130px" circle />
        <Image src="cat3.png" width="160px" height="130px" outline />
      </>
    );
    expect(ImageGroup).toMatchSnapshot();
  });
});
