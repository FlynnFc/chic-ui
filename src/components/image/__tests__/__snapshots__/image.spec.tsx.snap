// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Component Renders Correctly 1`] = `
Array [
  <img
    className="sc-bdfBQB kREmQd"
    height="130px"
    src="cat1.png"
    width="160px"
  />,
  <img
    className="sc-bdfBQB jbhbsW"
    height="130px"
    src="cat2.png"
    width="160px"
  />,
  <img
    className="sc-bdfBQB hgqmGJ"
    height="130px"
    src="cat1.png"
    width="160px"
  />,
  <img
    className="sc-bdfBQB beIrHm"
    height="130px"
    src="cat2.png"
    width="160px"
  />,
  <img
    className="sc-bdfBQB fBXjEW"
    height="130px"
    src="cat3.png"
    width="160px"
  />,
]
`;
